src/GitHub/Data/Reactions.hs

```diff
@@ -0,0 +1,39 @@
+-----------------------------------------------------------------------------
+-- |
+-- License     :  BSD-3-Clause
+--
+module GitHub.Data.Reactions where
+
+import GitHub.Data.URL         (URL)
+import GitHub.Internal.Prelude
+import Prelude ()
+
+data Reactions = Reactions
+    { reactionsUrl      :: !(Maybe URL)
+    , reactionsCount    :: !(Maybe Int)
+    , reactionsPlus1    :: !(Maybe Int)
+    , reactionsMinus1   :: !(Maybe Int)
+    , reactionsLaugh    :: !(Maybe Int)
+    , reactionsHooray   :: !(Maybe Int)
+    , reactionsConfused :: !(Maybe Int)
+    , reactionsHeart    :: !(Maybe Int)
+    , reactionsRocket   :: !(Maybe Int)
+    , reactionsEyes     :: !(Maybe Int)
+    }
+  deriving (Show, Data, Typeable, Eq, Ord, Generic)
+
+instance FromJSON Reactions where
+    parseJSON = withObject "Reactions" $ \o -> Reactions
+        <$> o .:? "url"
```

Comment:
Can we do `. .:? key .!= 0` or is there some difference between `Just 0` and `Nothing`?

Reply:
No, there is nothing signified by the Nothing case.  Can fix.